app/templates/inbox/irg-archive.hbs

```diff
@@ -4,10 +4,11 @@
   <table.menu as |menu|>
     <menu.general>
       <AuToolbar class="au-o-box" @size="large" as |Group|>
-        <Group>
+        <Group class="au-c-toolbar__group--vertical">
           <AuHeading @skin="2">
             {{t 'inbox.irgArchive.pageTitle'}}
           </AuHeading>
+          <h2>{{t 'inbox.irgArchive.pageSubTitle'}}</h2>
```

Comment:
Preferably use appuniversum components, so AuHeading here

skin 5 looks about right to me
https://appuniversum.github.io/ember-appuniversum/?path=/story/components-content-auheading--component&args=level:2;skin:5

Reply:
done